test/parallel/test-net-better-error-messages-port-hostname.js

```diff
@@ -3,12 +3,18 @@ const common = require('../common');
 const net = require('net');
 const assert = require('assert');
 
-const c = net.createConnection(common.PORT, '***');
+const s = net.createServer();
+s.listen(0);
+const port = s.address().port;
```

Comment:
@Trott We are asserting that the port is zero, right? That makes sure that no arbitrary port is assigned. We can probably leave a comment explaining it.